models/us_census_cbp/code/upload.py

```
@@ -0,0 +1,90 @@
+        )
+    return n
+
+
+def main():
+    only = sys.argv[1:] or TABLES
+    for table_slug in TABLES:
+        if table_slug not in only:
+            continue
+        try:
+            upload_table(table_slug)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)